spec/ndx-microscopy.namespace.yaml

```diff
@@ -8,6 +8,8 @@ namespaces:
   - cody.baker@catalystneuro.com
   - alessandra.trapani@catalystneuro.com
   schema:
-  - namespace: core
+  - namespace:
+    - core
+    - ndx-ophys-devices
```

Comment:
I was told we would need something like this

@rly does this look right? 

w.r.t. https://github.com/catalystneuro/ndx-ophys-devices